guide/onboarding/automatic-cloud-backup.md

```diff
@@ -95,7 +95,7 @@ By not making users manually backup, you decrease the risk of self-inflicted los
 
 While this is an easier way for users to set up a wallet, it brings in the potential threat of malicious third-party access. If a user’s cloud account is hacked or they forget their cloud login information, their funds could be compromised.
 
-Additionally, by bringing cloud storage providers into the equation, we assume that users trust their providers. This also raises the question of [decentralization]({{ 'guide/foundations/principles/#decentralization' | relative_url }}): a guiding principle of the bitcoin space and could be considered at conflict with the popular saying, “not your keys, not your coins”.
+By bringing cloud storage providers into the equation, we assume that users trust their providers. This also raises the question of [decentralization]({{ 'guide/foundations/principles/#decentralization' | relative_url }}): a guiding principle of the Bitcoin space and could be considered in conflict with the popular saying, “not your keys, not your coins”.
```

Comment:
I know photon is kind of on hold but could it recommended to designers to integrate some kind of encryption on the data stored in the cloud? Ideally by a users entered PIN as that is much harder to forgot than a passphrase.

Reply:
I updated the gallery image to use PIN-based auth 👍